### PR TITLE
[doc] Update feature documentation with new behaviours

### PR DESCRIPTION
**Pull request by yannis-rossetto**
This is part of story #21418 close artifacts with Done semantic

Some screenshots are also updated to match the updated UI.

### CODE REVIEW

File: languages/en/user-guide/code-versioning/gitlab.rst [hunk 1]
```diff
@@ -116,6 +116,35 @@ To reference GitLab tag, you have to use the keyword ``gitlab_tag`` followed by
 
 When you click on this reference, you will be redirected to your GitLab instance, on the page displaying the commit tagged with ``<tag_name>``.
 
+Close Tuleap artifact with GitLab commit messages
+`````````````````````````````````````````````````
+
+When integrating a GitLab repository in a Tuleap project, there is an option (disabled by default) that allow to close artifacts in this project based on GitLab commit messages.
+If a commit message containing a closure keyword is pushed in a GitLab repository integrated in a Tuleap project and this integration has activated this feature,
```
Comment (LeSuisse) [inline languages/en/user-guide/code-versioning/gitlab.rst:123]: I suggest we use the terms "closing pattern" to match what the [GitLab documentation is using](https://docs.gitlab.com/ee/user/project/issues/managing_issues.html#closing-issues-automatically) in this section. It's likely to cause less confusion to GitLab users and it avoid the usage of a mostly UK word (the US equivalent is cloture).

File: languages/en/user-guide/code-versioning/gitlab.rst [hunk 2]
```diff
@@ -116,6 +116,35 @@ To reference GitLab tag, you have to use the keyword ``gitlab_tag`` followed by
...
 
+Close Tuleap artifact with GitLab commit messages
+`````````````````````````````````````````````````
+
+When integrating a GitLab repository in a Tuleap project, there is an option (disabled by default) that allow to close artifacts in this project based on GitLab commit messages.
+If a commit message containing a closure keyword is pushed in a GitLab repository integrated in a Tuleap project and this integration has activated this feature, 
+then the artifact will be closed :
+
+* If the artifact is in the same project of the integration
+* If a :ref:`status semantic <status-semantic>` is defined for this artfact
```
Comment (LeSuisse) [inline languages/en/user-guide/code-versioning/gitlab.rst:127]: Do not we need a Done semantic?

**Review by LeSuisse — CHANGES_REQUESTED** [review]

File: languages/en/user-guide/code-versioning/gitlab.rst (hunk 1 above)
Comment (yannis-rossetto) [reply languages/en/user-guide/code-versioning/gitlab.rst:123]: Thanks for the suggestion, I will modify the content

File: languages/en/user-guide/code-versioning/gitlab.rst (hunk 2 above)
Comment (yannis-rossetto) [reply languages/en/user-guide/code-versioning/gitlab.rst:127]: I forgot to add this, thanks.

File: languages/en/user-guide/code-versioning/gitlab.rst [hunk 3]
```diff
@@ -143,7 +143,7 @@ Some variations of these keywords are handled:
 * ``Closing``/``Fixing``/``Resolving``/``Implementing``
 
 The artifact will be closed by a Tuleap bot named ``Tracker Workflow Manager`` and a comment will be added to add context about this closure.
-The first closed value retrieved by Tuleap will be used in this closure.
+The first done value retrieved by Tuleap will be used in this closure. If there is no done semantic defined, then the first closed value will be used.
```
Comment (LeSuisse) [inline languages/en/user-guide/code-versioning/gitlab.rst:146]:
```suggestion
The first done value retrieved by Tuleap will be used to close the artifact. If there is no done semantic defined, then the first closed value will be used.
```

File: languages/en/user-guide/code-versioning/gitlab.rst [hunk 4]
```diff
@@ -120,15 +120,15 @@ Close Tuleap artifact with GitLab commit messages
...
 then the artifact will be closed :
 
 * If the artifact is in the same project of the integration
-* If a :ref:`status semantic <status-semantic>` is defined for this artfact
+* If a :ref:`done semantic <done-semantic>` or a :ref:`status semantic <status-semantic>` is defined for this artfact
 * If the artifact is not yet closed
 * If the push occurs in the GitLab repository's default branch
 
-The handled closure keywords (case insensitive) are:
+The handled closure keywords (case insensitive) in closing patterns are:
```
Comment (LeSuisse) [inline languages/en/user-guide/code-versioning/gitlab.rst:131]:
```suggestion
The following keywords (case insensitive) can be used to close an artifact:
```

File: languages/en/user-guide/code-versioning/gitlab.rst [hunk 5]
```diff
@@ -120,15 +120,15 @@ Close Tuleap artifact with GitLab commit messages
 `````````````````````````````````````````````````
 
 When integrating a GitLab repository in a Tuleap project, there is an option (disabled by default) that allow to close artifacts in this project based on GitLab commit messages.
-If a commit message containing a closure keyword is pushed in a GitLab repository integrated in a Tuleap project and this integration has activated this feature, 
+If a commit message containing a closing pattern is pushed in a GitLab repository integrated in a Tuleap project and this integration has activated this feature, 
 then the artifact will be closed :
 
 * If the artifact is in the same project of the integration
-* If a :ref:`status semantic <status-semantic>` is defined for this artfact
+* If a :ref:`done semantic <done-semantic>` or a :ref:`status semantic <status-semantic>` is defined for this artfact
```
[the remark this answers is not in the document]
Comment (yannis-rossetto) [reply languages/en/user-guide/code-versioning/gitlab.rst:127]: Done

File: languages/en/user-guide/code-versioning/gitlab.rst (hunk 3 above)
Comment (yannis-rossetto) [reply languages/en/user-guide/code-versioning/gitlab.rst:146]: Done

File: languages/en/user-guide/code-versioning/gitlab.rst (hunk 4 above)
Comment (yannis-rossetto) [reply languages/en/user-guide/code-versioning/gitlab.rst:131]: Done

**Review by LeSuisse — APPROVED** [review]

**Pull request merged**